widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -178,7 +178,7 @@ export function TokenList(props: PropTypes) {
                 start={
                   <ImageSection>
                     <Image src={token.image} size={30} />
-                    {isTokenPinned(token) && (
+                    {isTokenPinned(token, props.type) && (
```

Comment:
When the wallet is not connected, the pin tokens come above the others, but when the wallet is connected, the sortedTokens function works by balance.
![Screenshot from 2023-11-21 16-43-23](https://github.com/rango-exchange/rango-client/assets/123297941/ca0891af-487d-43e1-9481-bdb31373f80d)
![Screenshot from 2023-11-21 16-43-19](https://github.com/rango-exchange/rango-client/assets/123297941/daf4663f-b117-4fc2-b96d-e6ff808b46fd)